app/models/user.rb

```diff
@@ -12,6 +12,9 @@ class User < ApplicationRecord
   has_many :donations, dependent: :destroy
 
   validates :email, uniqueness: true
+  validates_presence_of :first_name, :last_name, :email
```

Comment:
Double check this works with facebook login if an email is missing. It's not guaranteed that email is sent over and it not does this model invalidate when trying to save?

Reply:
Yeah, it invalidates. I didn't know the facebook email wasn't guaranteed to come through. I can remove the email from this validation for now.

Reply:
I just attempted to remove the email from the presence validation but it seems like it's still being validate. Do you have any idea why? I think it might have something to do with devise, but I'm not quite sure.

Reply:
I just removed the explicit user email presence validation. I don't know how the devise facebook login was setup, but I can look into later and try to figure out how to deal with that.

Reply:
Not sure, @rashadmad and @gcwokeup worked on that. Any ideas guys?